CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ### Features
 
+- `[jest-environment-jsdom, jest-environment-node]` Add default
+  `exportConditions` for both `jsdom` and `node` environments
+  ([#11924](https://github.com/facebook/jest/pull/11924))
```

Comment:
```suggestion
- `[jest-environment-jsdom, jest-environment-node]` [**BREAKING**] Add default `exportConditions` for both `jsdom` and `node` environments ([#11924](https://github.com/facebook/jest/pull/11924))
```

Reply:
Why do you consider it a breaking change? It just add a couple new default conditions that already exist in Node.js itself...

Reply:
because a test that previously pulled in one file can now pull in another changing the behaviour

Reply:
I see... yes, makes sense. It was anyway a broken test, but yes, it's a behaviour change and would break.

Reply:
yep! more correct, but still a breaking change 😀